bower.json.erb

```diff
@@ -12,7 +12,7 @@
     "jquery-ujs": "*",
     "jquery-migrate": "3.0.*",
     "bind-polyfill": "^1.0.0",
-    "yeast": "1.11.0",
+    "yeast": "https://github.com/moneyadviceservice/yeast.git#10985_rad_MAPS-Banner",
```

Comment:
This will have to be updated to reflect the latest version of Yeast once it is deployed